README.md

```diff
@@ -1,3 +1,48 @@
 # NOT READY
 
 will eventually be [Reason](https://reasonml.github.io) bindings to [React Router](https://reacttraining.com/react-router/)
+
+## Install
+
+```bash
+npm install --save bs-react-router
+```
+
+Then add bs-react-router to bs-dependencies in your bsconfig.json:
+
+```bash
+{
+  ...
+  "bs-dependencies": ["bs-react-router"]
+}
+```
+
+## Example
```

Comment:
Suggestion: we can move the examples to actual `*.re` files, you know for sure then if it compiles ;)

E.g. https://github.com/reasonml-community/bs-downshift/tree/master/examples

Reply:
Yes this is a very good point ! Good catch !